uint8 STATIC_RED = 1
uint8 STATIC_GREEN = 2
uint8 STATIC_BLUE = 3
uint8 STATIC_CYAN = 4
uint8 STATIC_PURPLE = 5
uint8 STATIC_YELLOW = 6
uint8 STATIC_WHITE = 7
uint8 LOW_FLASH_RED = 8
uint8 LOW_FLASH_GREEN = 9
uint8 LOW_FLASH_BLUE = 10
uint8 LOW_FLASH_CYAN = 11
uint8 LOW_FLASH_PURPLE = 12
uint8 LOW_FLASH_YELLOW = 13
uint8 LOW_FLASH_WHITE = 14
uint8 FAST_FLASH_RED = 15
uint8 FAST_FLASH_GREEN = 16
uint8 FAST_FLASH_BLUE = 17
uint8 FAST_FLASH_CYAN = 18
uint8 FAST_FLASH_PURPLE = 19
uint8 FAST_FLASH_YELLOW = 20
uint8 FAST_FLASH_WHITE = 21

uint8 effect_id
---
bool success
string message
